appveyor.yml

```diff
@@ -36,20 +46,19 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
-    - cmd: conda update conda
-    - cmd: conda config --add channels conda-forge --force
-    - cmd: conda config --system --add pinned_packages defaults::conda
+    - cmd: conda.exe config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    - cmd: conda.exe update conda
+    - cmd: conda.exe config --add channels conda-forge --force
```

Comment:
This removes the `defaults` pin that @jswhit added in https://github.com/Unidata/netcdftime/commit/42ccc0566aba5016cc957d1f04c06b7d8611a40f and keep using `conda-forge`.

I don't have any strong opinion on this for `netcdftime` but I'd love to revert this in netcdf4-python (PR soon) b/c conda-forge usually has newer `hdf5` and `netcdf-c` and they are compiled with extra flags/options, so it is better to test against that.

Reply:
+1  - please do submit a pull request for netcdf4-python.  I really don't know what I'm doing with appveyor - so I'd much rather have your solution.